README.md

```diff
@@ -1,7 +1,9 @@
-Orleans - Distributed Actor Model
+<p align="center">
+  <img src="https://github.com/dotnet/orleans/blob/gh-pages/assets/logo_full.png" alt="Orleans logo" width="600px"> 
+</p>
```

Comment:
Does the centering of the badges also center the remaining text? I don't any change to the badge code in #2017, but I see a difference in the image you posted.
![image](https://cloud.githubusercontent.com/assets/203839/17403819/b2a4824c-5a9c-11e6-9949-8b61f291b72b.png)